src/app/api/team-member/route.js

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { NextResponse } from "next/server";
+import { PrismaClient as prisma } from "@prisma/client";
+
+const healthCheck = async (req, res) => {
+  return NextResponse.json({ status: "ok" });
+};
+
+const getTeamAllMembers = async (req, res) => {
+  try {
+
+    const { data } = prisma.teamMember.findMany();
+    return new NextResponse.json(data);
+
+  } catch (error) {
+
+    return new NextResponse.error(error);
+
+  }
+};
+
+const addTeamMember = async (req, res) => {
+  try {
+
+    const { data } = NextResponse.json(req.body);
+    prisma.teamMember.create({ data });
+    
+  } catch (error) {
+
+    console.log(error);
+    return new NextResponse.error(error);
+    
+  }
+};
+
+export { healthCheck as GET, getTeamAllMembers as GET, addTeamMember as POST };
```

Comment:
```suggestion
export { healthCheck as PUT, getTeamAllMembers as GET, addTeamMember as POST };
```

2 functions can have same name hence either use PUT or any other allowed name ([link](https://nextjs.org/docs/app/building-your-application/routing/route-handlers#supported-http-methods))

Anyways, you can add health check to team-member endpoint instead of team-member/id endpoint